src/AppInstallerCLICore/Workflows/InstallFlow.h

```diff
@@ -136,17 +136,45 @@ namespace AppInstaller::CLI::Workflow
     // Outputs: None
     void ReportIdentityAndInstallationDisclaimer(Execution::Context& context);
 
-    // Installs a specific package installer. See also InstallSinglePackage & InstallMultiplePackages.
+    // Installs a specific package installer. See also InstallSinglePackage & ProcessMultiplePackages
     // Required Args: None
     // Inputs: InstallerPath, Manifest, Installer, PackageVersion, InstalledPackageVersion?
     // Outputs: None
     void InstallPackageInstaller(Execution::Context& context);
-
-    // Manages the dependencies for a single package.
+    
+    // Installs the dependencies for a specific package.
     // Required Args: None
+    // Inputs: InstallerPath, Manifest, Installer, PackageVersion, InstalledPackageVersion?
+    // Outputs: None
+    struct InstallDependencies : public WorkflowTask
+    {
+        InstallDependencies(
+            bool createDependencySubContexts) :
+            WorkflowTask("InstallDependencies"),
+            m_createDependencySubContexts(createDependencySubContexts) {}
+
+        void operator()(Execution::Context& context) const override;
+
+    private:
+        bool m_createDependencySubContexts = false;
+    };
+
+    // Downloads all of the package dependencies of a specific package.
+    // Required Args: none
     // Inputs: Manifest, Installer
-    // Outputs: InstallerPath
-    void ManageDependencies(Execution::Context& context);
+    // Outputs: None
+    struct DownloadPackageDependencies : public WorkflowTask
+    {
+        DownloadPackageDependencies(
+            bool includeInstalledPackages):
+            WorkflowTask("DownloadPackageDependencies"),
+            m_includeInstalledPackages(includeInstalledPackages) {}
+
+        void operator()(Execution::Context& context) const override;
+
+    private:
+        bool m_includeInstalledPackages = false;

```

Comment:
nit: this may not be needed as it's always set to true because only winget download flow use it